web-ui/view/step.ml

```diff
@@ -52,3 +52,87 @@ let title_card ~status ~card_title ~hash_link ~created_at ~finished_at
             rebuild_button;
           ];
       ])
+
+let log_highlight_js =
+  Tyxml.Html.script ~a:[]
+    (Tyxml.Html.Unsafe.data
+       {|"
```

Comment:
I think you just want the `{|` delimiter and not the opening quote. For quoted string literals, the delimiters are just `{|` and `|}`, if you put a quote it'll be part of the string content.